client/src/modules/Schedule/components/TableView/TableView.tsx

```diff
@@ -57,6 +62,7 @@ const getColumns = ({
       filters: TAGS.map(status => ({ text: renderTagWithStyle(status.value, tagColors), value: status.value })),
       defaultFilteredValue: tagFilter,
       filtered: tagFilter.length > 0,
+      filteredValue: tagFilter || null,
```

Comment:
Similar to thing above: `tagFilter` is type of `string[]`, how is it supposed to be `null`?